ansible/vars/topo_msft_four_asic_vs.yml

```diff
@@ -0,0 +1,197 @@
+ASIC0:
+  topology:
+    ASICs:
+      ASIC2:
```

Comment:
rename this to NEIGH_ASIC or PEER_ASIC? it is clearer

Reply:
Modified to NEIGH_ASIC